go-apps/meep-mon-engine/server/mon-engine.go

```diff
@@ -632,7 +632,7 @@ func removeExpectedPods(sandboxName string) {
 		// Get sandbox-specific pod name
 		var podName string
 		prefix := "meep-"
-		sandboxPrefix := prefix + sandboxName + "-"
```

Comment:
Destroying any sandbox will remove the sandbox pods from the list of expected sandbox pods to monitor. This will prevent the mon engine from reporting missing/crashing sandbox pods in other sandboxes.

We need to have sandbox-specific keys in the expected pods list. I.e. continue to mangle the sandbox name in the key, however use the expected pod status name to store the expected release name without the mangled sandbox name.

Reply:
Resolved